_posts/2024-08-06-czi-eoss6-announcement.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "Chan Zuckerberg Initiative considers scikit-learn an Essential Open Source Software"
+date: August 6, 2024
+categories:
+  - Funding
+tags:
+  - Open Source
+  - Funding
+  - Internship
+  - Diversity
+featured-image: sklearn_czi.png
+
+postauthors:
+  - name: Guillaume Lemaitre
+    website: https://github.com/glemaitre
+    image: guillaume-lemaitre.jpg
+  - name: Lucy Liu
+    website: https://github.com/lucyleeow
+    image: lucyliu.jpeg
+---
+<div>
+  <img src="/assets/images/posts_images/{{ page.featured-image }}" alt="">
+  {% include postauthor.html %}
+</div>
+
+We are delighted to announce that `scikit-learn` is awarded a grant from
+the [Chan Zuckerberg Initiative (CZI)](https://chanzuckerberg.com/)'s [Essential Open
+Source Software for Science
+(EOSS)](https://chanzuckerberg.com/rfa/essential-open-source-software-for-science/)
+program. This grant is funded by [Wellcome Trust](https://wellcome.org/).
+As in previous round, this cycle supports open-source software projects that are
+essential to biomedical research. This is the third time that CZI EOSS supports
+`scikit-learn`. In previous rounds, the following projects were funded:
+
+- [CZI EOSS 1](https://chanzuckerberg.com/eoss/proposals/scikit-learn-maintenance-and-enhancement-for-gradient-boosting/)
+  helped at creating to the
+  [`HistGradientBoostingClassifier`](https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.HistGradientBoostingClassifier.html) and
+  [`HistGradientBoostingRegressor`](https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.HistGradientBoostingRegressor.html) estimators.
+  These estimators are the equivalent of gradient boosting models implemented in
+  `LightGBM` and `XGBoost`.
+- [CZI EOSS 4](https://chanzuckerberg.com/eoss/proposals/maintenance-extension-of-scikit-learn-machine-learning-in-python/)
+  extended `scikit-learn` to work better with missing values and categorical data in
+  several estimators.
+
+Both grants allowed us to maintain and enhance `scikit-learn` to better serve the
+community.
+
+In this new grant, we will focus on improving the evaluation and inspection of
+predictive models. We provide in the next section a detailed description of the
+proposed work.
+
+## Predictive models evaluation & inspection
+
+When building a machine learning pipeline for a specific research problem, two key
+aspects are closely connected: (i) design the pipeline and (ii) assess, analyze, and
+inspect it. Researchers strive to identify the optimal pipeline, maximizing specific
+evaluation metrics, while also seeking at explaining the validity and rationale behind
+the pipeline's predictions. This is the corner stone to properly answering research
```

Comment:
```suggestion
the pipeline's predictions. This is the cornerstone of answering research
```

I think it is "cornerstone of", if we want to say "... to" we need something like "key to". I'd stick with cornerstone. Removed "properly" because it didn't read great somehow and I couldn't think of a way to shuffle things around